repl/src/main/scala/ammonite/repl/interp/ClassLoaders.scala

```diff
@@ -121,11 +123,42 @@ class SpecialClassLoader(parent: ClassLoader, parentHash: Array[Byte])
     * classloader can get at them.
     */
   val newFileDict = mutable.Map.empty[String, Array[Byte]]
+  val classOutputDir = System.getProperty("ammonite.repl.class.outputdir", null)
```

Comment:
I tried to look at the code but I got a bit lost.  Could you help me with this change?

I see `replArgs: Seq[Bind[_]]` in the constructor.  Did you want me to use this parameter?  I dunno how SpecialClassLoader can access this parameter, though.